### ISSUE TITLE
Vulnerability reported  image aspnet:5.0 from https://hub.docker.com/_/microsoft-dotnet-aspnet

### ISSUE THREAD

**Issue by ZhiqingWan** [dotnet/dotnet-docker#3089]
### Describe the Bug
The issue that we are encountering is that we have downloaded an image aspnet:5.0 from https://hub.docker.com/_/microsoft-dotnet-aspnet. 
The libfetch library from the independent package; 3.13:apk-tools:2.12.5-r0 contains an out-of-bounds vulnerability as per screenshot attached. 
We would like to know if Microsoft will be releasing a new image to address this vulnerability? And what will be the ETA?

### Steps to Reproduce
![image](https://user-images.githubusercontent.com/89833390/131459947-84e0479a-ec83-4919-8c85-e17e9a5420b2.png)

**Comment by mthalman** [comment]
Can you be more specific about the tag you are referencing?  The `mcr.microsoft.com/dotnet/aspnet:5.0-alpine` and `mcr.microsoft.com/dotnet/aspnet:5.0-alpine3.13` tags are up-to-date and contain the necessary fix to address this vulnerability:

```console
❯ docker pull mcr.microsoft.com/dotnet/aspnet:5.0-alpine
5.0-alpine: Pulling from dotnet/aspnet
540db60ca938: Already exists
a224d43c06e9: Already exists
ae1f97804ccb: Pull complete
c653c4e9f2de: Pull complete
Digest: sha256:ce1fa12d71192d2ee5918688ae73ede8d3c921045caa04d6bae14575da85ea63
Status: Downloaded newer image for mcr.microsoft.com/dotnet/aspnet:5.0-alpine
mcr.microsoft.com/dotnet/aspnet:5.0-alpine

❯ docker run --rm mcr.microsoft.com/dotnet/aspnet:5.0-alpine /bin/sh -c "apk list apk-tools"
apk-tools-2.12.7-r0 x86_64 {apk-tools} (GPL-2.0-only) [installed]
```

Notice that in your screenshot the fix is available starting in version 2.12.6-r0 and the image contains version 2.12.7-r0.

Please verify you've pulled the latest version of the image.

**Comment by MichaelSimons** [comment]
[Triage] Closing as this vulnerability has already been fixed in the latest images.
